dsl_parser/tests/test_functions.py

```diff
@@ -21,6 +21,31 @@
 from dsl_parser.tests.abstract_test_parser import timeout
 
 
+class TestGetInput(AbstractTestParser):
```

Comment:
should add additional test for when the value returned isnt a number (something that breaks)